target/linux/mediatek/image/filogic.mk

```diff
@@ -643,14 +643,19 @@ define Device/glinet_gl-x3000-xe3000-common
   DEVICE_PACKAGES := kmod-mt7915e kmod-mt7981-firmware mt7981-wo-firmware mkf2fs \
     kmod-fs-f2fs kmod-hwmon-pwmfan kmod-usb3 kmod-usb-serial-option \
     kmod-usb-storage kmod-usb-net-qmi-wwan uqmi
+  IMAGES += factory.bin
+  IMAGE/factory.bin := append-kernel | pad-to 32M | append-rootfs
```

Comment:
Why do you add a factory image? Is this needed for OpenWrt?

Reply:
Cargo cult I'm afraid, to match what was done when adding the GL-MT6000 (Flint 2) u-boot support.

Reply:
ok, this should not be needed for the u-boot. Maybe the vendor bootloader of the GL-MT6000  needs a special image. It is probably better to leave it our in this PR.

Reply:
ok, @mrkiko  thinks it is needed, maybe leave it in.

Reply:
Great okay, as requested it's still there in the rebased version I pushed.